liquidity/liquidity.go

```diff
@@ -22,6 +23,12 @@ import (
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
+const (
+	// DefaultFailureBackOff is the default amount of time we wait after
+	// a swap has failed on a channel to recommend using it again.
+	DefaultFailureBackOff = time.Hour * 24 * 7
```

Comment:
A week sounds like a very long time. Maybe the default should be a bit shorter, perhaps 1 day. Or is this to prevent suggesting the same bad channel every day?

Reply:
The thought here is that we don't want to get into a state where we have one really unbalanced channel, but it actually isn't possible to loop out through it. Thinking about it a bit more, I think we want some kind of exponential backoff here. WDYT about decreasing this to one day and adding a todo for more intelligent backoff?

Reply:
Yes, that sounds good to me!